src/compute/src/render/reduce.rs

```diff
@@ -1152,75 +1181,77 @@ where
     /// `validating` indicates whether we want this stage to perform error detection
     /// for invalid accumulations. Once a stage is clean of such errors, subsequent
     /// stages can skip validation.
-    fn build_bucketed_negated_output<S, R>(
+    fn build_bucketed_negated_output<S, Tr>(
         &self,
         input: &Collection<S, (Row, Row), Diff>,
         aggrs: Vec<AggregateFunc>,
-    ) -> Collection<S, (Row, R), Diff>
+    ) -> Arranged<S, TraceAgent<Tr>>
     where
         S: Scope<Timestamp = G::Timestamp>,
-        R: MaybeValidatingRow<Row, Row>,
+        Tr::ValOwned: MaybeValidatingRow<Row, Row>,
+        Tr: Trace
+            + for<'a> TraceReader<
+                Key<'a> = DatumSeq<'a>,
+                KeyOwned = Row,
```

Comment:
Same here: `KeyOwned` doesn't need to be specified.

Reply:
Fixed here now!